src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -1523,8 +1523,37 @@ object ResolverType {
   case object Aether extends ResolverType("aether") {
     override def optionsDoc: Option[Doc] = None
   }
-  case object Coursier extends ResolverType("coursier") {
-    override def optionsDoc: Option[Doc] = None
+  case class Coursier(
+    hashInHttpHeaders: Option[Boolean]
+  ) extends ResolverType("coursier") {
+    def getHashInHttpHeaders: Boolean = hashInHttpHeaders.getOrElse(false)
+
+    override def optionsDoc: Option[Doc] = {
+
+      val items = List(
+        (
+          "hashInHttpHeaders",
```

Comment:
I'm not really sure why we wouldn't always want to enable this if it works. Can you think of a reason? Can you document the reason here so someone reading the code can remember.

Reply:
Unfortunately, it doesn't work with all Maven repository servers. E.g., https://repo1.maven.org/maven2/ only supplies SHA-1 checksums, so it would be a waste to enable this option if that's the repo that's getting used.

Reply:
makes sense.